packages/components/src/common/Notifications/NotificationIcon/NotificationIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+
+import { BannerColor } from "../../../Banner";
+import CheckCircleIcon from "../../../Icons/CheckCircle";
+import DangerousRoundedIcon from "../../../Icons/custom/DangerousRounded";
+import ForumIcon from "../../../Icons/Forum";
+import NotificationsIcon from "../../../Icons/Notifications";
+import WarningIcon from "../../../Icons/Warning";
+import styles from "./NotificationIcon.module.css";
+
+type Props = {
+  /**
+   * Which variant to use. Each variant has a unique icon and color.
+   */
+  variant: BannerColor;
+};
+
+export const variantToIconAssetsMap = {
+  brand: {
+    icon: NotificationsIcon,
+    style: styles.iconBrand,
+    title: "attention",
+  },
+  neutral: {
+    icon: ForumIcon,
+    style: styles.iconNeutral,
+    title: "notice",
+  },
+  success: {
+    icon: CheckCircleIcon,
+    style: styles.iconSuccess,
+    title: "success",
+  },
+  warning: {
+    icon: WarningIcon,
+    style: styles.iconWarning,
+    title: "warning",
+  },
+  alert: {
+    icon: DangerousRoundedIcon,
+    style: styles.iconAlert,
+    title: "alert",
+  },
+};
+
+/**
+ * An icon that appears in notifcation components (Banner and Toast).
+ * This icon communicates the tone of the notification and is through its image,
+ * color, and title text.
+ */
+const NotificationIcon = ({ variant }: Props) => {
```

Comment:
curious why you switched to `variant` instead of `color`?

Reply:
It already kinda bugs me that `Banner` and `Toast` have a prop called `color` that also controls which icon is used, but it felt especially weird to use `color` in the situation where it primarily controls which icon is returned. But I don't care that much and am fine with changing it to `color` if consistency is more important.

Reply:
mmm yep makes sense